atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -41,7 +41,7 @@
 import com.palantir.common.base.FunctionCheckedException;
 
 public class CassandraClientPoolTest {
-    public static final int POOL_REFRESH_INTERVAL_SECONDS = 10;
+    public static final int POOL_REFRESH_INTERVAL_SECONDS = 3 * 60;
```

Comment:
That is, some of the tests now take more than 10 seconds (e.g. "make sure we retry 6 times before failing when getting socket timeouts") where before we didn't have to worry about these scheduled jobs interfering.